EventFilter/L1TRawToDigi/plugins/L1TValidationEventFilter.cc

```diff
@@ -95,22 +91,15 @@ bool
 L1TValidationEventFilter::filter(edm::Event& iEvent, const edm::EventSetup& iSetup)
 {
   using namespace edm;
-  
-  edm::Handle<FEDRawDataCollection> feds;
-  iEvent.getByToken(fedData_, feds);
 
-  if (!feds.isValid()) {
-    LogError("L1T") << "Cannot unpack: no FEDRawDataCollection found";
+  Handle<int> triggerCount;
+  iEvent.getByLabel(InputTag(src_.label(),"triggerCount"), triggerCount);
```

Comment:
The reason for having it configurable is that if you write a different producer for it, you may have a different label.

Since the most common use case is to set the input tag in a cfi fie (or even in the fillDescriptions method) and always use the default, users won;t have to type the extra label anyway.